data/json/itemgroups/art_antiques_crafts.json

```
@@ -206,6 +206,7 @@
     "subtype": "distribution",
     "entries": [
       { "item": "helmet_army_outdated", "prob": 5 },
+      { "item": "helmet_soviet_tanker", "prob": 5 },
```

```suggestion
      { "item": "helmet_soviet_tanker", "prob": 2 },
```
Reduced by about half here.